ClientLibrary/lib/build.gradle

```diff
@@ -29,6 +35,7 @@ archivesBaseName = "face"
 version = "1.4.3"
 
 // Upload artifacts to maven central repository staging servers
+
```

Comment:
[Nit] Remove this.

Reply:
Yes, that shouldn't have reached the pull request